src/traces/bar/hover.js

```diff
@@ -67,15 +67,15 @@ module.exports = function hoverPoints(pointData, xval, yval, hovermode) {
 
     if(trace.orientation === 'h') {
         pointData.x0 = pointData.x1 = xa.c2p(di.x, true);
-        pointData.xLabelVal = di.s;
+        pointData.xLabelVal = di.b + di.s;
 
         pointData.y0 = ya.c2p(barPos(di) - barDelta, true);
         pointData.y1 = ya.c2p(barPos(di) + barDelta, true);
         pointData.yLabelVal = di.p;
     }
     else {
         pointData.y0 = pointData.y1 = ya.c2p(di.y, true);
-        pointData.yLabelVal = di.s;
+        pointData.yLabelVal = di.b + di.s;
```

Comment:
@n-riesco this line here is breaking hover for `barmode: 'stack'` bar charts. 

Can share some insights on why you added this?